api/src/main/kotlin/io/datalbry/precise/api/schema/field/BasicFieldTypes.kt

```diff
@@ -36,7 +36,7 @@ enum class BasicFieldTypes(val id: String) {
     /**
      * sequence of 8-bit unsigned bytes
      */
-    BYTES("bytes"),
+    BYTE("byte"),
```

Comment:
Is this now a sequence of bytes, or a byte?

Reply:
Just one byte - the type may be multi valued which would result into a byte sequence. (Basically ByteArray...) 

Going to change the javadoc